iroh-resolver/src/unixfs_builder.rs

```diff
@@ -738,7 +738,7 @@ fn add_blocks_to_store_chunked<S: Store>(
 ) -> impl Stream<Item = Result<AddEvent>> {
     let mut chunk = Vec::new();
     let mut chunk_size = 0u64;
-    const MAX_CHUNK_SIZE: u64 = 1024 * 1024 * 16;
+    const MAX_CHUNK_SIZE: u64 = 1024 * 1024;
```

Comment:
why?

Reply:
The http2 transport is reusing using the http2 framing. That way you don't have to add a framing layer on top, and they are specifically made to allow user level framing.

But - the default max frame size is pretty small. It is no problem to increase it, but that means a larger per connection buffer. Not the end of the world in the RPC case, but something you definitely want to avoid in a P2p case where you have lots of connections.

So, the current default is 1024 * 1024 plus margin. I just adjusted it here because once this is merged I want to get rid of the PutMany thing anyway, and replace it with a call where the blocks are streamed from the client, as I wanted to do in the first place.

We will have to think about how big this should be. I think bitswap allows a maximum block size of 4MB, but unixfs won't produce such large blocks.

Reply:
And yes, I guess quic-rpc needs to make this configurable. But I think we can leave that for later.

Reply:
Made an issue: https://github.com/n0-computer/quic-rpc/issues/11